config/schedule.yml

```diff
@@ -3,13 +3,12 @@
 
 # executed At 12:00 on every day-of-month.
 internal_check_new_versions_job:
-  cron: "0 12 */1 * *"
-  class: "Internal::CheckNewVersionsJob"
+  cron: '0 12 */1 * *'
+  class: 'Internal::CheckNewVersionsJob'
   queue: scheduled_jobs
 
 # executed At every 5th minute..
 trigger_scheduled_items_job:
-  cron: "*/5 * * * *"
-  class: "TriggerScheduledItemsJob"
+  cron: '*/1 * * * *'
```

Comment:
should we change this back to 5 minutes?  or we could choose something in between and update the comment above.

Reply:
We can change it back to 5 min.